relion/protocols/protocol_classify3d.py

```diff
@@ -68,8 +68,14 @@ def _setSamplingArgs(self, args):
             args['--offset_range'] = self.offsetSearchRangePix.get()
             args['--offset_step'] = self.offsetSearchStepPix.get() * self._getSamplingFactor()
 
+            # check if sigma_ang is in extra params
+            # before adding the default value
             if self.localAngularSearch:
-                args['--sigma_ang'] = self.localAngularSearchRange.get() / 3.
+                if self.extraParams.hasValue():
```

Comment:
the last param "--sigma_ang" if added in extra options will anyway overwrite whatever was specified before, but I guess I can live with this change

Reply:
What you say is true, relion works without the change but complains and prints a warning in the Output Log tab. 
with this code the warning is not printed.